articles/_posts/2013-10-01-designing-a-brand-in-the-open.md

```diff
@@ -6,7 +6,9 @@ tags:
 author: garthdb
 priority: 0.6
 ---
-We have already started some open design discussion for this site in [GitHub Issues][github-issues] ([IA][ia] and [UI][ui]).  This discussion will lead to some loose design prototypes in a new repo that the community can fork and contribute to, but for the project branding we've decided to try [LayerVault][layervault] and [Creative Cloud][creative-cloud]. You do not need to sign up for an account on Creative Cloud to comment there.
+**Update:** we are switching to LayerVault to [Pixelapse][pixelapse]. We've had issues with LayerVault and our files were actually lost. Our experience with Pixelapse has been quite positive so far.
```

Comment:
Can you switch "to" to "from". 
So it will read we are switching from LayerVault to Pixelapse